javaauth/test/marathon_auth_plugin_test.py

```diff
@@ -0,0 +1,422 @@
+#!/usr/bin/python
+
+from __future__ import print_function
+import unittest
+import os
+import socket
+import argparse
+import json
+import sys
+import httplib
+import urllib
+import requests
+import base64
+import subprocess
+import time
+
+def post_data(url,headers,payload):
+    result = requests.post(url, data=json.dumps(payload), headers=headers)
+    return (result.status_code)
+
+def get_data(url,headers,payload):
+    url = url+payload['id']
+    result = requests.get(url,data=payload,headers=headers)
+    return result.status_code
+
+def put_data(url,headers,payload):
+    url = url+payload['id']
+    result = requests.put(url, data=json.dumps(payload), headers=headers)
+    return (result.status_code)
+
+def delete_data(url,headers,payload):
+    url = url+payload['id']
+    result = requests.delete(url, data=json.dumps(payload), headers=headers)
+    return (result.status_code)
+
+def store_original_remote_file():
+    try:
+        user = os.getlogin()
+        remote_path = "/data/marathon/plugins/user-permissions.json"
+        hostname1 = user+"@daldevmesoszk01"
+        subprocess.call(['scp', ':'.join([hostname1,remote_path]),'/tmp'])
+    except:
+        print("Failed to store original file from remote server")
+
+def scp_to_remote_location(filepath):
+    try:
+        user = os.getlogin()
+        remote_path = "/data/marathon/plugins/user-permissions.json"
+        hostname1 = user+"@daldevmesoszk01"
+        hostname2 = user+"@daldevmesoszk02"
+        hostname3 = user+"@daldevmesoszk03"
```

Comment:
Removed the hardcoded values. The machine names are now passed as an parameter in the command line.